101-signalr/azuredeploy.json

```diff
@@ -67,14 +93,58 @@
           {
             "flag": "ServiceMode",
             "value": "[parameters('serviceMode')]"
+          },
+          {
+            "flag": "EnableConnectivityLogs",
+            "value": "[parameters('enableConnectivityLogs')]"
+          },
+          {
+            "flag": "EnableMessagingLogs",
+            "value": "[parameters('enableMessagingLogs')]"
           }
-        ]
+        ],
+        "cors": {
+          "allowedOrigins": "[parameters('allowedOrigins')]"
+        },
+        "networkACLs": {
+          "defaultAction": "deny",
+          "publicNetwork": {
+            "allow": [
+              "ClientConnection"
+            ]
+          },
+          "privateEndpoints": [
+            {
+              "name": "mySignalRService.1fa229cd-bf3f-47f0-8c49-afb36723997e",
+              "allow": [
+                "ServerConnection"
+              ]
+            }
+          ]
+        },
+        "upstream": {
+          "templates": [
+            {
+              "categoryPattern": "*",
+              "eventPattern": "connect,disconnect",
+              "hubPattern": "*",
+              "urlTemplate": "https://example.com/chat/api/connect"
+            }
+          ]
+        }
       },
+      "kind": "SignalR",
       "sku": {
         "capacity": "[parameters('capacity')]",
         "name": "[parameters('pricingTier')]"
       },
       "type": "Microsoft.SignalRService/SignalR"
     }
-  ]
+  ],
+  "outputs": {
+    "SignalRPrimaryConnectionString": {
```

Comment:
The reason?  Because we'd like to show our customers how to retrive SignalR connection string using ARM template. We got several customer tickets for this

Reply:
Because outputs are available to anyone with read permission at scope... this is not a pattern we should be demonstrating with customers (it creates Sev 1's)

Reply:
Got it. Thanks. Removed.